user-manual/metadata/METS.rst

```diff
@@ -0,0 +1,182 @@
+LASTMODDATE.
+
+.. code:: xml
+
+  <metsHdr CREATEDATE=”2019-06-18T23:52:11” LASTMODDATE=”2019-06-27T00:28:56”/>
+
+
+.. _dmdsec:
+
+<dmdSec>
+^^^^^^^^
+
+There will be one dmdSec for the whole AIP, which contains PREMIS Intellectual
+Entity information.
+
+.. literalinclude:: scripts/METS.xml
+   :language: xml
+   :lines: 15-35
+
+There may also be one or more dmdSecs for descriptive metadata. The dmdSecs are
+numbered dmdSec_1, dmdSec_2, etc. and contain Dublin Core metadata as a default.
+
+.. literalinclude:: scripts/METS.xml
+   :language: xml
+   :lines: 64-91
+
+MDTYPE will be **DC** for Dublin Core metadata; if another metadata standard is
+included then the MDTYPE will be **OTHER**. 
+
+If the user does not enter any DC metadata during transfer/ingest and no DC
+metadata was included with the transfer (e.g. as a metadata.csv file, see
+:ref:`Import metadata <import-metadata>`), there will be no additional dmdSecs.
+
+
+.. _amdsec:
+
+<amdSec>
+^^^^^^^^^
+
+There is one amdSec for each object and wraps PREMIS entities as follows:
+
+* PREMIS Object: techMD (tehnical metadata)
+* PREMIS Event: digiprovMD (digital provenance metadata)
+* PREMIS Agent: digiprovMD (digital provenance metadata)
+* PREMIS Rights: rightsMD (rights metadata)
+
+Each amdSec will include one techMD. The techMD is used to wrap PREMIS Object
+semantic units, such as objectIdentifier, objectCharacteristics and
+originalName. See the example techMD below:
+
+.. literalinclude:: scripts/METS.xml
+   :language: xml
+   :lines: 212-258
+
+Each amdsec will include multiple digiprovMDs for PREMIS events related to the
+PREMIS Object. Below is an example of a PREMIS event for a fixity check:
+
+   .. literalinclude:: scripts/METS.xml
+      :language: xml
+      :lines: 820-854
+
+An amdSec for an original object may also contain one or more rightsMDs. The
+rightsMD may contain a reference to metadata in another file, such as rights.csv
+uploaded with a transfer. Below is an example of a PREMIS Rights statement in
+rightsMD section of the AIP METS file:
+
+.. literalinclude:: scripts/METS.xml
+   :language: xml
+   :lines: 2073-2108
+
+An amdSec will also have one ore more sourceMD sections. This is the only
+non-PREMIS section in an amdSec and no XML schema is used for the contents.
+SourceMD is created from the contents of the bag-info.txt whenever a zipped or
+unzipped bag transfer type is used. It is also created for a disk image transfer
+if the user entered meteadata about the disk image in the Transfer tab.
+
+
+.. _filesec:
+
+<fileSec>
+^^^^^^^^^
+
+There is one fileSec listing all files. The fileSec is organized into two or
+more of the following fileGrps:
+
+* original
+* preservation
+* service
+* submissionDocumentation
+* metadata
+* license
+* text/ocr
+* deleted
+
+These are known as USE attributes and must accompany each fileGrp. The fileGrp
+original is required for all METS files. If the AIP includes normalized files
```

Comment:
This is an interesting one. We normalize on two occasions so should we say specifically `normalized for preservation` so we aren't opening a question about `normalized for access`?

Reply:
good point. Best to be clear!